README.md

```diff
@@ -1,4 +1,14 @@
-# smarthome-localhomesdk
+# Local Home SDK Virtual Environment
+
+A mock Local Home Platform implemented as stubs for the Local Home SDK
+
+## Usage
```

Comment:
That would be ideal, yeah.  For now I think the functionality in this PR can fit within that goal.  For example, the tests here are still handy for testing this project.  I'll just need to expose some API (see issue #7) and reformat a couple things.  I opened issue #21 to specifically package everything as a library and will implement it in a separate PR.

Reply:
Another way to look at it would be that the scope of the project could start smaller (just be a library w/ examples) before becoming something more (a end-to-end testing tool).

The advantage starting with the former is that would already have a useful artifact (albeit, less friendly and turn-key that your end goal), the disadvantage is that you maybe to make significant change to it as you progress and try to make it fit with your end goal.
The advantage starting with the later is that you have something that fit exactly your end goal but that will get harder and harder to extract over time.

Choose you poison :) :test_tube: :skull:

Reply:
Hmm, your first suggestion (library w/ examples) sounds like the current way I'm looking at it.  The stubs and mocks in `common` are intended to be my "library" and the test I have is an example of its usage.

It's not too user-friendly yet, since the bundle needs to be placed in the project for example.  But as I flesh out the stubs I can move everything  to be exported as a node package and provide a formal library.

I don't _think_ I'm pigeonholing myself to the end-goal with this PR, since the test I currently have isn't going to be part of the end UX.  I'm using it mainly for development and as an example to make sure my stubs and mocks in `common/` function as intended.